### PR TITLE
added a basic splash screen, which goes for 3s

### PR DESCRIPTION
**Pull request by musfique113**
- splash screen added

### CODE REVIEW

**Comment by RafatMeraz** [comment]
Pull last update from my main branch and update your PR.

File: h20/lib/presentation/app.dart [hunk 1]
```diff
@@ -7,13 +8,16 @@ class H2OApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      title: 'Splash Screen',
       theme: ThemeData(
-        brightness: Brightness.light
+        primarySwatch: Colors.blue,
       ),
-      darkTheme: ThemeData(
```
Comment (RafatMeraz) [inline h20/lib/presentation/app.dart:13]: We have to maintain in both theme.

Comment (musfique113) [reply h20/lib/presentation/app.dart:13]: Okay, Working on it.

File: h20/pubspec.yaml [hunk 2]
```diff
@@ -59,9 +60,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
-  #   - images/a_dot_ham.jpeg
+  assets:
+    - images/
```
Comment (RafatMeraz) [inline h20/pubspec.yaml]: Never make separate folders for images/fonts/animation at the root level. Create an assets folder that will contain all types of child folders by type.

Comment (musfique113) [reply h20/pubspec.yaml]: I was aware of that information, but I simply forgot while working on this project. Thanks.

**Comment by musfique113** [comment]
I have added the necessary changes , kindly review and verify if everything is in order.

**Pull request merged**